h2o-extensions/xgboost/src/test/java/hex/tree/xgboost/XGBoostTest.java

```diff
@@ -24,6 +25,7 @@
 import java.io.PrintWriter;
 import java.util.Arrays;
 import java.util.HashMap;
+import java.util.Map;
 
 import static water.util.FileUtils.locateFile;
 
```

Comment:
It is still testing on a single node cluster, is is on purpose ? `stall_till_cloudsize(1);`